doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -0,0 +1,657 @@
+#############
+
+In child/parent images, the application controlled all images, so variables without a prefix would apply to the main application only.
+In Sysbuild, elements like file suffixes, shields, and snippets without an image prefix will be applied **globally** to all images.
+To apply them to a single image, they must be prefixed with the image name.
+Without doing this, projects with multiple images (for example, those with MCUboot) might fail to build due to invalid configuration for other images.
+
++-------------------------------+----------------------------------+-----------------------+
+| Configuration parameter       | Child/parent                     | Sysbuild              |
++===============================+==================================+=======================+
+| ``-DFILE_SUFFIX=...``         | Applies to main application only | Applies to all images |
++-------------------------------+----------------------------------+-----------------------+
+| ``-D<image>_FILE_SUFFIX=...`` | Applies to <image> only                                  |
++-------------------------------+----------------------------------+-----------------------+
+| ``-DSNIPPET=...``             | Applies to main application only | Applies to all images |
++-------------------------------+----------------------------------+-----------------------+
+| ``-D<image>_SNIPPET=...``     | Applies to <image> only                                  |
++-------------------------------+----------------------------------+-----------------------+
+| ``-DSHIELD=...``              | Applies to main application only | Applies to all images |
++-------------------------------+----------------------------------+-----------------------+
+| ``-D<image>_SHIELD=...``      | Applies to <image> only                                  |
++-------------------------------+----------------------------------------------------------+
+
+Configuration values that specify Kconfig fragment or overlay files (for example, ``EXTRA_CONF_FILE`` and ``EXTRA_DTC_OVERLAY_FILE``) cannot be applied globally using either child/parent image or sysbuild.
+They function the same in both systems:
+
+* Without a prefix, they will be applied to the main application only.
+
+* With a prefix, they will apply to that specific image only.
+
+.. _child_parent_to_sysbuild_migration_building:
+
+Building with sysbuild
+######################
+
+With |NCS| 2.7, building with ``west`` will use sysbuild by default in |NCS| repositories.
+To maintain compatibility with child/parent images, out-of-tree applications will not use sysbuild by default.
+Twister does not inherit this change and will not use sysbuild unless tests are updated.
+Similarly, CMake will not configure projects using sysbuild unless the invocation command is updated.
+
+.. note::
+
+    Sysbuild can optionally be set as the default when using west, this means that unless west is ran with ``--no-sysbuild`` then it will always use sysbuild
```

Comment:
```suggestion
    Sysbuild can optionally be set as the default when using west.
    This means that unless west is ran with ``--no-sysbuild`` then it will always use sysbuild
```